packages/corvid-e2e-tests/test/sanity.e2e.js

```diff
@@ -67,4 +68,44 @@ describe("browser sanity", () => {
       await openEditorCliCommand.waitForCommandToEnd();
     })
   );
+
+  test("should show a warning message when closing with unsaved changes and allow to continue working", async () => {
+    process.env.SKIP_UNSAVED_DIALOG = true;
+    const testEditorUrl = testSites[0].editorUrl;
+    const cliCloneCommandResult = await cliDriver.clone({
+      editorUrl: testEditorUrl
+    });
+    const editorCloneDriver = await connectToLocalEditor(
+      cliCloneCommandResult.editorDebugPort
+    );
+
+    const loginDriver = await editorCloneDriver.waitForLogin();
+    await loginDriver.login(getCorvidTestUser());
+
+    await cliCloneCommandResult.waitForCommandToEnd();
+
+    const openEditorCliCommand = await cliDriver.openEditor();
+    const editorEditDriver = await connectToLocalEditor(
```

Comment:
naming of the different drivers is really confusing

Reply:
made some changes